[RUN-8862] addedd to RN and files (#352)
run-ai/docs: docs/Researcher/user-interface/workspaces/create/create-ds.md

To create an NFS data source, provide:

* A data source name.

> * A Run:ai project scope. Choose one or more items from the tree. The NFS data source is assigned to one or more projects in the scope.

Reviewer: ```suggestion
* In the *Scope* pane, choose one item from the tree. The compute resource is assigned to that item and all its subsidiaries.
```

To create a host path data source, provide:

* A data source name.

> * A Run:ai project scope. Choose one or more items from the tree. The host path data source is assigned to one or more projects in the scope.

Reviewer: ```suggestion
* In the *Scope* pane, choose one item from the tree. The compute resource is assigned to that item and all its subsidiaries.
```
